### PR TITLE
Source Zendesk Chat: Implement Incremental sync

### LINKED ISSUE #3112: Source Zendesk Chat: Implement Incremental sync
## Tell us about the problem you're trying to solve
Implement Incremental sync for `read` method.

_Sub-task for Issue_ #1316

### PR DESCRIPTION
**Pull request by yevhenii-ldv**
## What
*closes #3112.*

## Pre-merge Checklist
- [ ] *Run integration tests*
- [ ] *Publish Docker images*

## Recommended reading order
1. `test.java`
1. `component.ts`
1. the rest

### CODE REVIEW

File: airbyte-integrations/connectors/source-zendesk-chat/integration_tests/integration_test.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+BASE_DIRECTORY = Path(__file__).resolve().parent.parent
+
+
+@pytest.fixture(name="config_credentials")
+def config_credentials_fixture():
+    with open(str(BASE_DIRECTORY / "secrets/config.json"), "r") as f:
+        return json.load(f)
+
+
+@pytest.fixture(name="configured_catalog")
```
Comment (keu) [inline airbyte-integrations/connectors/source-zendesk-chat/integration_tests/integration_test.py:41]: ❤️

File: airbyte-integrations/connectors/source-zendesk-chat/integration_tests/integration_test.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+def configured_catalog_fixture():
+    return ConfiguredAirbyteCatalog.parse_file(BASE_DIRECTORY / "sample_files/agent_timeline_catalog.json")
+
+
+class TestZendeskChatSource:
```
Comment (keu) [inline airbyte-integrations/connectors/source-zendesk-chat/integration_tests/integration_test.py:46]: we don't need this kind of tests if we use acceptance tests, please check HubSpot repo for example

File: airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/api.py [hunk 3]
```diff
@@ -80,12 +73,117 @@ def get_stream_data(self, response_data: Any) -> List[dict]:
...
+        if isinstance(value, int):
+            value = pendulum.from_timestamp(value / 1000.0)
+        elif isinstance(value, str):
+            value = pendulum.parse(value)
+        else:
+            raise ValueError(f"Unsupported type of datetime field {type(value)}")
+        return value
+
+
+class ZendeskChatTimeIncrementalStream(ZendeskChatBaseIncrementalStream, ABC):
```
Comment (keu) [inline airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/api.py:103]: there is no need to mention Incremental in the name, also I think it is quite redundant to mention ZendeskChat, but it is up to you

**Review by keu — CHANGES_REQUESTED** [review]
small comments

**Review by keu — APPROVED** [review]
let's address the acceptance tests issue and the rest is LGTM

File: README.md [hunk 4]
```diff
@@ -7,7 +7,7 @@
 **Data integration made simple, secure and extensible.**  
 The new open-source standard to sync data from applications, APIs & databases to warehouses, lakes & other destinations.
 
-[![](docs/.gitbook/assets/deploy-locally.svg)](docs/deploying-airbyte/on-your-workstation.md) [![](docs/.gitbook/assets/deploy-on-aws.svg)](docs/deploying-airbyte/on-aws-ec2.md) [![](docs/.gitbook/assets/deploy-on-gcp.svg)](docs/deploying-airbyte/on-gcp-compute-engine.md)
+[![](docs/.gitbook/assets/deploy-locally.svg)](docs/deploying-airbyte/local-deployment.md) [![](docs/.gitbook/assets/deploy-on-aws.svg)](docs/deploying-airbyte/on-aws-ec2.md) [![](docs/.gitbook/assets/deploy-on-gcp.svg)](docs/deploying-airbyte/on-gcp-compute-engine.md)
```
Comment (sherifnada) [inline README.md:10]: make sure to pull master into both this branch and the base branch so these changes don't show up (guessing you didn't make this change)

**Review by sherifnada — CHANGES_REQUESTED** [review]
can you pull master into base branch so the diff is cleaned?

**Review by keu — APPROVED** [review]

**Pull request merged**